quinn-proto/src/connection/streams.rs

```diff
@@ -416,18 +429,23 @@ impl Streams {
 
     /// Cease accepting data on a stream
     ///
-    /// Returns `true` iff the peer should be notified.
-    pub fn stop(&mut self, id: StreamId) -> Result<bool, UnknownStream> {
+    /// Returns structure which indicates whether this requires transmissing
```

Comment:
`transmissing` is not a word, I think, and "whether this requires transmitting and additional frames" also seems wrong? Maybe "and" -> "any"?